src/llmcompressor/transformers/finetune/session_mixin.py

```diff
@@ -286,7 +293,7 @@ def compute_loss(
 
         # TODO: do we need these model signature columns?
         inputs = {k: inputs[k] for k in inputs if k in self._signature_columns}
-        loss = super().compute_loss(model, inputs, return_outputs=return_outputs)
+        loss = super().compute_loss(model, inputs, return_outputs, num_items_in_batch)
```

Comment:
It would be nice to keep named args here

Reply:
I’d prefer to keep the call more concise in the case that the reader knows exactly what the function call is, unless there are any other good reasons

Reply:
I think we should use arg names to help with readability and in the event that one of them is removed, the error will indicate that it is no longer relevant